fabric-access-v1/src/main/resources/fabric-access-v1.accesswidener

```diff
@@ -0,0 +1,11 @@
+accessWidener	v1	named
+
+accessible    method    net/minecraft/entity/SpawnRestriction    register    (Lnet/minecraft/entity/EntityType;Lnet/minecraft/entity/SpawnRestriction$Location;Lnet/minecraft/world/Heightmap$Type;Lnet/minecraft/entity/SpawnRestriction$SpawnPredicate;)V
```

Comment:
```suggestion
transitive-accessible    method    net/minecraft/entity/SpawnRestriction    register    (Lnet/minecraft/entity/EntityType;Lnet/minecraft/entity/SpawnRestriction$Location;Lnet/minecraft/world/Heightmap$Type;Lnet/minecraft/entity/SpawnRestriction$SpawnPredicate;)V
```
Need the prefix so projects that depend on this module can actually access them :P.

Reply:
It's needed for all actual AW ?

Reply:
I guess, yeah? You don't need to prefix it if it is just for internal use.
I suggested one change just for example.